infra/docker/testground-prometheus/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM prom/prometheus
```

Comment:
We spoke about this, and I was reluctant to adding yet another moving piece that the user had to remember to build/make. I see you've decided to go this way anyway. What are the reasons? Shouldn't this be accompanied with a change in the `Makefile`?

Reply:
We don't make this with the Makefile, I have EnsureImage building this at runtime. This is different than the bind mount, but this way we don't have to ask the user to place a config file in a specific location.

We could probably actually move the local docker sidecar to this method, and have nothing to build with the Makefile.

I should have left this open for review, rather than after the fact. I've been in a "just merge it" mode, this week, sorry I didn't give enough opportunity to review before,